Tallies/TallyClerks/collisionClerk_class.f90

```diff
@@ -178,17 +182,26 @@ end function getSize
   !!
   !! See tallyClerk_inter for details
   !!
-  subroutine reportInColl(self, p, xsData, mem)
+  subroutine reportInColl(self, p, xsData, mem, virtual)
     class(collisionClerk), intent(inout)  :: self
     class(particle), intent(in)           :: p
     class(nuclearDatabase), intent(inout) :: xsData
     type(scoreMemory), intent(inout)      :: mem
+    logical(defBool), intent(in)          :: virtual
     type(particleState)                   :: state
     integer(shortInt)                     :: binIdx, i
     integer(longInt)                      :: adrr
     real(defReal)                         :: scoreVal, flx
     character(100), parameter :: Here =' reportInColl (collisionClerk_class.f90)'
 
+    ! Calculate flux sample based on physical or virtual collision
+    if (self % virtual) then
+      flx = ONE / xsData % getMajorantXS(p)
+    else
+      if (virtual) return
+      if (.not. virtual) flx = ONE / xsData % getTotalMatXS(p, p % matIdx())
```

Comment:
```suggestion
      flx = ONE / xsData % getTotalMatXS(p, p % matIdx())
```
Since we know that if virtual is true we `return` in the previous line I don't think this `if` is necessary.